### DISCUSSION TITLE
trPC request session is undefined

### DISCUSSION THREAD

**Discussion by tanyongkuan** [t3-oss/create-t3-turbo#1085] [General]
I'm trying to startup this repo. I have push the DB to supabase.

pnpm run dev, everything is working fine. But I got trPC request from rsc by undefined which I realised it came from nextAuth.

I also tried deploying the auth-proxy as a vercel function but didnt work out as well. I have configured to have the same secret and vercel url with /r

Anyone can guide me in this?

**Comment by Zeff01** [comment]
Same error here! Have you solve this already? @tanyongkuan
